### PR TITLE
when sso login is enable just show link to show login form

### PR DESCRIPTION
**Pull request by harshkothari410**

### CODE REVIEW

File: view/authentication/login/login.pug [hunk 1]
```diff
@@ -22,18 +22,29 @@ html
...
+          br
+          a#show-login(href='#') Login Using Username and Password
+          div.local-form.hide#login-form   
+            form(role='form', name='loginform')
+              label#errorInfo
+              .form-group
+                input.form-control(type='text', name="username", placeholder='Enter username' required)
+              .form-group
+                input.form-control(type='password', name="password", placeholder='Password' required)
+              button.btn.btn-primary.btn-block(type='submit') Submit
```
Comment (iamigo) [inline view/authentication/login/login.pug]: While we're here, can you also just comment out the "forgot password" button until we actually implement that functionality?

Comment (harshkothari410) [inline view/authentication/login/login.pug]: Yes I can do that. Doing it.

**Comment by harshkothari410** [comment]
I started the review app you can test it @pallavi2209 @annyhe

File: view/authentication/login/login.pug (hunk 1 above)
Comment (harshkothari410) [inline view/authentication/login/login.pug]: @iamigo updated it :)

**Comment by pallavi2209** [comment]
@harshkothari410 Can you please setup sso the way you did in previous PR so that I can check the sso enabled case as well.

**Comment by iamigo** [comment]
@pallavi2209 @harshkothari410 https://en.wiktionary.org/wiki/give_a_man_a_fish_and_you_feed_him_for_a_day;_teach_a_man_to_fish_and_you_feed_him_for_a_lifetime

**Comment by harshkothari410** [comment]
Added SSO to the App

**Comment by pallavi2209** [comment]
@harshkothari410 Congrats. You are an expert in setting up SSO now. :)
@iamigo True believer of the proverb here :)

**Comment by harshkothari410** [comment]
Yay !!! @pallavi2209 thanks for explaining everything. I agree @iamigo 👍

**Comment by iamigo** [comment]
@annyhe can you please also take a quick look at Harsh's UI changes here, and merge if approve? Thx!

**Comment by annyhe** [comment]
@iamigo No problem will do

**Comment by harshkothari410** [comment]
@annyhe can you merge this ?

**Comment by iamigo** [comment]
everything passing now, ok to merge @annyhe @pallavi2209 @harshkothari410 ?

**Comment by pallavi2209** [comment]
Fine with me.

**Pull request merged**